README.md

```diff
@@ -46,17 +46,47 @@ In file included from /usr/include/gtk-3.0/gtk/gtk.h:106:0,
 ```
 They are expected and don't produce any problem on the systray.
 
+Windows
+---------
+Download cygwin // https://cygwin.com/setup-x86_64.exe
+```
+Install with the necessary packages:
+
+mingw64-x86_64-gcc-core
+mingw64-x86_64-gcc-g++ 
+and
+x86_64-w64-mingw32-c++
+x86_64-w64-mingw32-gcc
+make
+
+Add to windowspath "C:\cygwin64\bin"
+```
+Build it
+```
+make build 
```

Comment:
For me this still not working, I've `win 10` +` cygwin64` getting:

```bash
$ make build
[build.sh] VENDOR_PATH = providers
[+] Using go in /cygdrive/c/dev/Go/bin/go [go version go1.15.4 windows/amd64]
[+] Building Go library with standard Go compiler
# 0xacab.org/leap/bitmask-vpn/pkg/backend
pkg\backend\init.go:108:47: undefined: bitmask.Autostart
pkg\backend\init.go:109:15: undefined: bitmask.NewAutostart
pkg\backend\init.go:112:16: undefined: bitmask.DummyAutostart
pkg\backend\status.go:46:18: undefined: bitmask.Autostart
make: *** [Makefile:94: lib/libgoshim.a] Error 2
```

Reply:
since we rebased from main with qt implementation, its also not working for me. but im still working on this branch and  heard that kali got worked main on win-10.

Reply:
hi @kyxap - are you running from the "main" branch? this error is related to a conditional compilation that disables autostart for windows, because the library was giving some trouble.

i'm indeed using the makefile to build from a win 10 install. there might be troubles with some hardcoded paths that probably can be better autodetected, but I'll try to improve it and for sure will accept patches with any fixes.

Reply:
hey @kalikaneko finally I was able to build. Will keep digging... thanks!

Reply:
hey @kyxap, nice that you are able to build! Can you maybe write your compiler- and toolset here?

Reply:
@ok-sh looks like I need add more clarification on what I meant :D 
I was able to build just one bin: `bitmask-connect` which I can run in Win 10 and basically get vpn working thought command line without ui.

But after some changing in makefiles I was able to build: `bitmask-vpn` which starts as systray and actually working somehow O_o. 

Anyway: 
I'm using cygwin64 `CYGWIN_NT-10.0 pc 3.1.7(0.340/5/3) 2020-08-22 17:48 x86_64 Cygwin` install link got from this VS doc: https://code.visualstudio.com/docs/cpp/config-mingw

Reply:
@kyxap - glad to hear!

Reply:
@ok-sh
FYI there MR in progress for cygwin64: https://0xacab.org/leap/bitmask-vpn/-/merge_requests/123